src/components/HeaderBlock/HeaderBlock.js

```diff
@@ -256,16 +250,20 @@ class HeaderBlock extends React.Component<Props> {
     if (item.user || item.userIcon) {
       return this.renderUser(!item.userIcon);
     }
-    if (item.title) {
+    if (item.title || item.customTitle) {
```

Comment:
Header block has already a way to have custom header - it was designed to accept various components in an array, so we could combine components in any way needed. 

Headers that need an icon in the middle of it, for example, could be created like this:
```headerProps={{ centerItems: [{ title: 'Send' }, { custom: (<ImageIcon source={lightningIcon} />) }, { title: symbol, color: baseColors.electricBlueIntense }] }}```

In such way we would not need an additional (almost single use) component.

Reply:
I'm sure I looked for this, but missed it somehow, thanks! 👍